foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model/OpenAiCompletionOutput.java

```diff
@@ -13,6 +14,7 @@
 @Accessors(chain = true)
 @EqualsAndHashCode
 @ToString
+@Beta
 public class OpenAiCompletionOutput {
```

Comment:
(Explanation)

<details><summary>This is the kind of warning visible to some of our users - depending on their IDE settings.</summary>

![image](https://github.com/user-attachments/assets/010c33b3-faaa-4042-84b7-6ef53de4a260)

</details>